templates/interstitial.html

```diff
@@ -8,6 +8,21 @@
   {% if block.content.title %}
     <h1>{{ block.content.title }}</h1>
   {% endif %}
-  {% set contents = block.content.contents %}
-  {% include 'partials/contents.html' %}
+
+  {% for item in block.content.contents %}
+    {%- if item.title -%}
+      <h2>{{item.title}}</h2>
+    {% endif %}
+    {%- if item.description -%}
+      <p>{{item.description}}</p>
+    {% endif %}
+    {%- if item.list -%}
+      <ul>
+        {%- for list_item in item.list -%}
+          <li>{{list_item}}</li>
+        {% endfor %}
+      </ul>
+    {% endif %}
+  {% endfor %}
```

Comment:
This is almost identical to what is in `contents.html`, the only thing that changes is the tag for `item.title`. I think we could just set a property like `title_type` which determines whether to use `h2` or a `strong` tag.

Reply:
I moved it to one file and "if" based on title type.